project/src/components/reviews-form/reviews-form.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState } from 'react';
+import RatingStarsRadioGroup from '../rating-stars-radio-group/rating-stars-radio-group';
+
+function ReviewsForm(): JSX.Element {
+  const [formState, setFormState] = useState({
+    rating: 0,
+    message: '',
+  });
+
+  return (
+    <form className="reviews__form form" action="/" method="post">
+      <RatingStarsRadioGroup label="Your review" onChange={(event) => setFormState({ ...formState, rating: Number(event.target.value) })} />
+      <textarea className="reviews__textarea form__textarea" id="review" name="review" placeholder="Tell how was your stay, what you like and what can be improved" defaultValue={formState.message} onChange={(event) => setFormState({...formState, message: event.target.value})} />
```

Comment:
Тут строчка тоже получилось очень большой

Предлагаю просто скомпоновать то, как  передаются пропсы

Это будет что-то вроде

```suggestion
      <textarea 
        className="reviews__textarea form__textarea" 
        id="review" 
        name="review" 
        placeholder="Tell how was your stay, what you like and what can be improved"
        ... и так далее
/>
```

И так далее